### ISSUE TITLE
Error: Unexpected close tag

### ISSUE THREAD

**Issue by bitfed** [theoephraim/node-google-spreadsheet#24]
Trying to use getRows to retrieve.

    var GoogleSpreadsheet = require('google-spreadsheet');

    var mySheet = new GoogleSpreadsheet('1X8c0vyB9syJ12c1Pjwj0RfCwJ4WIcW3VET0ZKDWyTFA');

    mySheet.getRows(1, function (err, row_data) {
      if (err) {
        console.error(err);
      } else {
        console.log(row_data);
      }
    });
Receiving this error:
`[Error: Unexpected close tag
Line: 920
Column: 9
Char: >]`

**Comment by ldhertert** [comment]
Just a heads up, I ran into this issue to and was able to get the code working by authenticating as specified in the readme.  That said, this sheet was set to "anyone with the link can view" so it shouldn't have required authentication.

**Comment by individual11** [comment]
Getting same issue, and don't want to put authentication info anywhere, so set the spreadsheet to anyone can edit.

**Comment by theoephraim** [comment]
There is a bit of a process to get sheets to be publicly accessible on google's API. See the readme: https://github.com/theoephraim/node-google-spreadsheet#a-note-on-authentication

I just did a quick test on a sheet I created with this ID: `1Gf1RL2FUjQpE6nJ4ywuX7hpZFqQ8oLE2yMAgzF7VsF0`
and everything works as expected.

The issue is that there are 2 steps to get your sheet public readable
- Set the sheet to "anyone can find and view"
![step1](https://www.evernote.com/shard/s133/sh/ec73ea23-cb1d-4b62-aa4f-fc2a4f3025e7/58de77829feacbe291e7140252e718c3/res/e72c2260-6f53-48cd-8fb3-70dca6f804a7/skitch.png?resizeSmall&width=832)

- Go to "File > Publish to the web" and click the "publish button
![step2](https://www.evernote.com/shard/s133/sh/d55e5b9d-9053-4361-bdc6-dfa92db68b52/7113e7838c0213df9b96b9a5cf566cb0/res/fd125114-e3fe-4af0-8364-34ddbdfae975/skitch.png?resizeSmall&width=832)

Maybe adding some better error messaging would help. Not sure if this situation can be easily detected, but I'll see what I can do.
